app/webpacker/controllers/modal_link_controller.js

```diff
@@ -30,4 +33,16 @@ export default class extends Controller {
   getIdentifier() {
     return "modal";
   }
+
+  #setPathToFormAction(confirmButton) {
+    const isSubmitButton = confirmButton.type === 'submit';
+    const path = this.modalDatasetValue['data-path'];
+
+    if(isSubmitButton && path){
+      const form = confirmButton.parentElement;
+      form.setAttribute('action', path);
+
+      delete this.modalDatasetValue['data-path'];
```

Comment:
This surprised me, but I suppose it's good practice to clean up because this controller is re-used for any product. I guess it enforces that the modal can only be used once (until you click on another delete link)